package.json

```diff
@@ -16,7 +16,7 @@
     "type": "git",
     "url": "https://github.com/usablica/intro.js"
   },
-  "main": "intro.js",
+  "main": "intro.es5.js",
```

Comment:
I'm not sure if changing the default main entrypoint is a good idea right now. Not entirely sure how this would affect people. Could you please revert this? You should still have access to the `intro.es5.js` in the npm package/CDNs.